tasties_app/tests/conftest.py

```diff
@@ -188,3 +188,61 @@ def signed_up_credentials():
 @pytest.fixture()
 def login_to_site(client):
     client.post('/login/', data={'username': VALID_USER, 'password': VALID_PASSWORD})
+
+
+@pytest.fixture
+def create_user():
+    """
+    This fixture create a user for test the recipe_view and the add_comment methods.
+    """
+    user = User(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
+    user.full_clean()
+    user.save()
+    return user
+
+
+@pytest.fixture
+def recipe_test():
```

Comment:
Again there's already a fixture `recipe` in conftest.py that creates a recipe. Bonus: creating a user by the name of "jakob" in a test fixture is in my opinion a dangerous thing to do (Remember what happened when initially saved data to the DB with migrations).